File: exs/mundo_1/c++/008.cpp
/*

008 - Conversor de medidas
  Escreva um programa que leia um valor em metros e o exiba convertido em centímetros e milímetros

*/

#include <iostream>

using namespace std;

int main() {
  double metros, centimetros, milimetros;
  cin >> metros;

  centimetros = (metros * 100);
  milimetros = (metros * 1000);

  cout << metros << " metros é igual a " << centimetros << " centimetros\n";
  cout << metros << " metros é igual a " << milimetros << " milimetros\n";

  return 0;
}

File: exs/mundo_1/ruby/008.rb
=begin
Desafio 008

Problema: Escreva um programa que leia um valor em metros e o
          exiba convertido em centímetros e milímetros.

Resolução do problema:
=end

print"Digite a medida em metros: "
medida = gets.chomp.to_f

cm = medida*100
mm = medida*1000

puts"#{medida} em centimetros é igual a #{cm}."
puts"#{medida} em milimetros é igual a #{mm}."
